app/src/main/java/com/example/b07group7project/ShopperNavigationActivity.java

```diff
@@ -21,7 +22,7 @@ protected void onCreate(Bundle savedInstanceState) {
         binding = ShopperNavigationActivityBinding.inflate(getLayoutInflater());
         setContentView(binding.getRoot());
 
-        homeFragment = new ExampleFragment();    // CHANGE THIS
+        homeFragment = new ItemPreviewFragment();    // CHANGE THIS
```

Comment:
When we finalize, I think the landing page would be the list of stores. This is fine for testing purposes though